bluepyentity/register.py

```diff
@@ -37,18 +37,63 @@ def _wrap_id(id_, type_):
 
 def _wrap_id_fetch(id_, forge):
```

Comment:
Not just for this line but for the whole commit ( 5b8ebda0e1124d5c3b6de96bde844b88de47ebf1 ). 
Currently:
* if a field is an expected id-field and is defined as a string
   * expect it to be an ID
   * raises if resource not found
* if it is a definition (=dict) but is missing an id, expect it needs to be registered
   * expect a full definition (don't try to parse with the Resource class below)
 
Some issues:
* If we want to parse the to-be-registered "subresource" with the Resource class, what to do there are several types defined?
* Some classes are "id-fields" but so far in general, they have not been registered and don't have ids
   * e.g.,`BrainLocation`
   * do we just register them?